saleor/graphql/checkout/mutations/checkout_customer_note_update.py

```diff
@@ -0,0 +1,57 @@
+import graphene
+
+from ....webhook.event_types import WebhookEventAsyncType
+from ...core import ResolveInfo
+from ...core.descriptions import ADDED_IN_321
+from ...core.doc_category import DOC_CATEGORY_CHECKOUT
+from ...core.mutations import BaseMutation
+from ...core.types import CheckoutError
+from ...core.utils import WebhookEventInfo
+from ...plugins.dataloaders import get_plugin_manager_promise
+from ..types import Checkout
+
+
+class CheckoutCustomerNoteUpdate(BaseMutation):
+    checkout = graphene.Field(Checkout, description="An updated checkout.")
+
+    class Arguments:
+        id = graphene.ID(
+            description="The checkout's ID." + ADDED_IN_321,
```

Comment:
Sorry, in the previous comment, I suggested changing it to ADDED_IN_321, but as this mutation is fully new, we don't need `ADDED_IN` in the arguments list. The one in `Meta.description` covers this.